src/routes/home/components/HomeFooter.tsx

```diff
@@ -20,7 +20,7 @@ type HomeFooterProps = {
 
 const HomeFooter = ({ navigateToImprint }: HomeFooterProps): ReactElement => (
   <Container>
-    <ContentSecondary>{'\u00A9'}LUNES2022</ContentSecondary>
+    <ContentSecondary>{'\u00A9'}LUNES2023</ContentSecondary>
```

Comment:
Maybe we can use `new Date().getFullYear()` instead so we don't have to update it every year.

Reply:
Thought about that as well, but didn't do it bc then in theory users can set their phone to some other year and maybe claim smth? But I am not an expert here if that would be a problem.

Reply:
Oooooh, interesting point. No idea 🤷